spec/system/placements/schools/mentors/remove_a_mentor_spec.rb

```diff
@@ -36,9 +36,9 @@
   context "when the mentor has placements" do
     before { create(:placement, mentors: [mentor_1], school:) }
 
-    scenario "User can not remove the mentor from the school" do
+    scenario "User cannot remove the mentor from the school" do
```

Comment:
Could you add a step to this test to check if you can see all of the mentors which are mentor to be displayed on this page?

Reply:
thanks, I've added an expectation to the then_i_see_i_cannot_removed_the_mentor method in latest commit.

Reply:
Would you mind splitting it into it's own step?
Something like `and_i_see_a_list_of_placements_assigned_to_mentor(mentor)`